bower.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "ifvisible.js",
+  "version": "1.0.2",
+  "main": "./src/ifvisible.js",
+  "ignore": {}
```

Comment:
The "ignore" bit? It specifies files not to install when bower installs the package, and for some reason is required. I'm not sure that many projects actually use it, but I guess in principle the idea would be that a project can ignore everything except their `dist` folder.

Also, I screwed up - that should be an empty array, not an empty object. Sorry, will fix it.

Over at https://github.com/bower/bower/issues/1388 you can see the references from commits where other people have just set this to empty arrays to silence the warning; that seems to be the normal approach.